client/src/App.tsx

```diff
@@ -5,7 +5,8 @@ import {
   Routes,
   Route,
 } from "react-router";
-import  { Home, Results, NotFound }  from './pages'
+import { getRestaurants, getRestaurant } from './utils/serverCalls';
```

Comment:
Where are they being called? When I comment out line 8 there are errors looking for Home, Results....

Reply:
It's probably because your branch is behind the current develop branch. You probably need to do a `git pull origin develop` while on your branch, then fix any merge conflicts that come up. I can walk you through it some other time if you want

Reply:
Okay I'm still not sure what to do here. Let me know if needs changes.

Reply:
To clarify, i believe we do need line 8 `import  { Home, Results, NotFound }  from './pages'` and the develop branch as it is now is still using that code.

Reply:
This comment was for this line, we do not need this in `App.tsx`:
```
import { getRestaurants, getRestaurant } from './utils/serverCalls';
```
not this line, we need this in `App.tsx`:
```

import  { Home, Results, NotFound }  from './pages'
```